tools/appimage.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+# The MIT License (MIT)
+# Copyright (c) 2016 Simon Peter
+
```

Comment:
Software being committed to this repos should me copyright Microsoft Corporation, per the CLA, and me already have a CLA.